openapi2conv/openapi2_conv.go

```diff
@@ -474,6 +475,23 @@ func ToV3SchemaRef(schema *openapi3.SchemaRef) *openapi3.SchemaRef {
 	for i, v := range schema.Value.AllOf {
 		schema.Value.AllOf[i] = ToV3SchemaRef(v)
 	}
+
+	schema = ToV3SchemaTypes(schema)
+
+	return schema
+}
+
+// ToV3SchemaTypes performs the type & format migrations from v2 to v3
+func ToV3SchemaTypes(schema *openapi3.SchemaRef) *openapi3.SchemaRef {
```

Comment:
you dont need to return the schema, its a pointer

Reply:
I know, it's their convension